CHANGELOG/CHANGELOG-1.7.0.md

```diff
@@ -8,6 +12,14 @@
     - [Features](#features)
     - [Bugs](#bugs)
 
+# v1.7.1
+
+## Changelog since v1.7.0
+
+## Changes by Kind
+
+- CSM 1.7.1 release specific changes. ([#911](https://github.com/dell/csm/issues/911))
```

Comment:
@VamsiSiddu-7 @Prabhu-Dell Are we not calling out what the changes are as discussed? GO Lang update to 1.20.5?